server/modules/amazon-client/myIdentity.js

```diff
@@ -0,0 +1,10 @@
+/* Copyright (c) Trainline Limited, 2016-2017. All rights reserved. See LICENSE.txt in the project root for license information. */
+
+'use strict';
+
+let AWS = require('aws-sdk');
+let memoize = require('modules/memoize');
+
+let myIdentity = memoize(() => (new AWS.STS()).getCallerIdentity().promise());
+
```

Comment:
In a way. My intent here was to make the call to the AWS API once and to do it lazily; the first time myIdentity is called, not when the module is loaded.